sycl/test/inline-asm/malloc_shared_32.cpp

```diff
@@ -58,12 +57,12 @@ int main() {
         svm_scatter.4.1 (M1, 16) %1.0 V53.0
     }
     )a" ::"rw"(&b[i]),
-                            "rw"(&b[i] + 16), "rw"(&a[i]), "rw"(&a[i] + 16), "rw"(&c[i]),
-                            "rw"(&c[i] + 16));
+                        "rw"(&b[i] + 16), "rw"(&a[i]), "rw"(&a[i] + 16),
+                        "rw"(&c[i]), "rw"(&c[i] + 16));
```

Comment:
These changes are also irrelevant.

Reply:
Yes, this is unrelated. Clang-format complained about this.  I will change this to the original.